templates/solutions/telco/_form-telco-5g-core.html

```diff
@@ -0,0 +1,94 @@
+<div class="p-modal" style="display: none;" id="telco-5g-core-modal">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="telco-5g-core-modal">
+    <div class="p-modal__header">
+      <button class="p-modal__close" aria-controls="data-kafka-modal" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
```

Comment:
Not sure why, but the modal won't close once opened. Tested on both chrome and ff

Reply:
Fixed, thanks for the catch!